docs/general/adapters.md

```diff
@@ -14,47 +43,89 @@ Doesn't follow any specific convention.
 
 ### JSON
 
-It also generates a json response but always with a root key. The root key **can't be overridden**, and will be automatically defined accordingly to the objects being serialized.
+The response document always with a root key.
+
+The root key **can't be overridden**, and will be derived from the resource being serialized.
+
 Doesn't follow any specific convention.
 
 ### JSON API
 
 This adapter follows **version 1.0** of the format specified in
```

Comment:
Should ref https://github.com/rails-api/active_model_serializers/blob/master/docs/jsonapi/schema.md somehow